packages/python/plotly/plotly/express/_doc.py

```diff
@@ -1,7 +1,9 @@
 import inspect
 
-colref = "(string: name of column in `data_frame`)"
-colref_list = "(list of string: names of columns in `data_frame`)"
+colref = "(string: name of column in `data_frame`, or array_like object)"
```

Comment:
worth adding that `int` column names are accepted also?

Reply:
done